docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -353,4 +353,68 @@ Example response:
         }
     ]
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+
+==== Assign or unassign users from alerts
+
+Allows you to assign and unassign users from alerts. 
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/assignees`
+
+====== Example request
+
+Assigns and unassigns users to alerts:
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/signals/assignees
+
+{
+  "assignees": {
+    "add": ["u_o4kzon2tUP0u189YjKVT0rTR_HBOED3JmyLLE6MrulY_0"],
+    "remove": ["u_P4HW8xg4_xRVI7Oa-i6Ys1Gxe7k3jqZteAeZe6ZctEc_0"]
+  },
+  "ids": [
+    "854f5eceeec1b4cd5495ad18c4259d6e5631a6677bc10c033edb318397d45459",
+    "00968e97805854d0aa356968cad971d5184cdf91ebd458720c5b4099f4a5229a"
+  ]
+}
+--------------------------------------------------
+// KIBANA
+
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+====== Response payload
+
+A JSON object containing the number of updated alerts.
+
+Example response
```

Comment:
```suggestion
Example response:
```

Reply:
Resolved in [205d2ed](https://github.com/elastic/security-docs/pull/4475/commits/205d2ed9d98447f666cf0d1ae3a5f45da6d90c78).